docs/content/docs/content/_index.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
This file should already exists in [docs/content/_index.md](https://github.com/apache/iceberg-docs/blob/main/docs/content/_index.md) and shouldn't need to be copied here. Changes can be made to the existing location and that will get updated as soon as the version branch is cut.